### PR TITLE
Update to avoid putting HA user in sudoers

### PR DESCRIPTION
**Pull request by DubhAd**
None of the steps require that HA is added to sudoers or given a password. Rewriting those steps to remove that, to improve overall security.

### CODE REVIEW

File: source/_docs/ecosystem/certificates/lets_encrypt.markdown [hunk 1]
```diff
@@ -474,37 +454,12 @@ $ crontab -e
  
  
 #### Option 2:
```
Comment (fabaff) [inline source/_docs/ecosystem/certificates/lets_encrypt.markdown]: Did you remove the whole section on purpose?

Comment (DubhAd) [reply source/_docs/ecosystem/certificates/lets_encrypt.markdown]: Yes, because the section is entirely about adding homeassistant to sudoers.

Comment (fabaff) [reply source/_docs/ecosystem/certificates/lets_encrypt.markdown]: Not according to the diff. `Option 2:` covers the renewal process through an automation rule.

**Comment by frenck** [comment]
@DubhAd Could you take a look at the feedback provided by @fabaff, that would be awesome. This PR is running a bit stale...

**Comment by DubhAd** [comment]
Yup, will do. Got distracted by other things.

**Comment by frenck** [comment]
Since there was no response at all, I'm declaring this PR stale and closing it.
Feel free to re-open this PR when you feel like that is the appropriate thing to do.

**Pull request closed without merging**